src/navigation.ts

```diff
@@ -824,9 +824,12 @@ export class Navigation {
    * @returns whether keyboard navigation is currently allowed.
    */
   canCurrentlyNavigate(workspace: Blockly.WorkspaceSvg) {
-    const accessibilityMode = workspace.isFlyout
-      ? workspace.targetWorkspace?.keyboardAccessibilityMode
-      : workspace.keyboardAccessibilityMode;
+    const accessibilityMode = (
+      workspace.getRootWorkspace() ??
+      workspace.targetWorkspace?.getRootWorkspace() ??
```

Comment:
These two are new. I'm assuming `targetWorkspace` is only used for flyouts which is why that fallback below is a correct logical replacement to the previous `isFlyout` check, but are these root workspace checks always correct and in this order?

It might help to add a line comment before this explaining when each condition can occur, and why the fallback is correct if more than 2 can occur at once (since `workspace` will always be present).

Reply:
That's correct, added a comment explaining the logic here.